examples/bridge-app/linux/main.cpp

```diff
@@ -285,6 +270,7 @@ int AddDeviceEndpoint(Device * dev, EmberAfEndpointType * ep, const Span<const E
             EmberAfStatus ret;
             while (1)
             {
+                DeviceLayer::StackLock lock;
```

Comment:
This is not going to work on Mac, fwiw.  It's a bad pattern; we should be queuing things to the right thread instead of doing that.

Reply:
Good point. 

This fixes the commands that add/delete endpoints for the bridge test plan ('2' and '4'). (Not directly related to changing from a switch to a temp sensor, but I noticed it was failing as I ran through the tests.)

Would it be better to keep the lock or remove the lock for now with a Todo comment, then change it to schedule the add/remove in a separate PR?

Reply:
It's fine to leave the lock for now, since it does help on Linux, but do a followup to do the scheduling.

Reply:
Thanks, I added a comment indicating the change is still needed.